Sources/AccountSDKIOSWeb/Client/ClientConfiguration.swift

```diff
@@ -17,6 +19,23 @@ public struct ClientConfiguration {
         case proNo = "https://payment.schibsted.no"
         case pre = "https://identity-pre.schibsted.com"
         case proDk = "https://login.schibsted.dk"
+        
+        var sessionService: String {
+            let sessionServiceStr: String
+            switch self {
+            case .proCom:
+                sessionServiceStr = "https://session-service.login.schibsted.com"
+            case .proFi:
+                sessionServiceStr = "https://session-service.login.schibsted.fi"
+            case .proNo:
+                sessionServiceStr = "https://session-service.payment.schibsted.no"
+            case .pre:
+                sessionServiceStr = "https://session-service.identity-pre.schibsted.com"
+            case .proDk:
+                sessionServiceStr = "https://session-service.login.schibsted.dk"
+            }
```

Comment:
@saltwoo is this the right urls for the different environment of session-service?

Reply:
yes, that is correct!